styleguide/derek/index.jade

```diff
@@ -25,11 +25,15 @@ block page
         h6 helpful links
         ul.nav.nav-pills.nav-stacked
           li
-            a(href="derek/examples") Examples
+            a(href="derek/solutions") Solutions
+          li
+            a(href="derek/templates") Templates
           li
             a(href="derek/development") Development
           li
             a(href="derek/workflow") Workflow/Process
```

Comment:
For these incomplete pages, should we have a `(Coming Soon)` or maybe just hide them for now, as the links to "Need to Add" pages are kinda annoying.